packages/migration/src/migrations/hearth/20240703161352-fix-practitioner-email.ts

```diff
@@ -0,0 +1,47 @@
+/*
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/.
+ *
+ * OpenCRVS is also distributed under the terms of the Civil Registration
+ * & Healthcare Disclaimer located at http://opencrvs.org/license.
+ *
+ * Copyright (C) The OpenCRVS Authors located at https://github.com/opencrvs/opencrvs-core/blob/master/AUTHORS.
+ */
+
+import { Db, MongoClient } from 'mongodb'
+
+const USER_MGNT_MONGO_URL =
+  process.env.USER_MGNT_MONGO_URL || 'mongodb://localhost/user-mgnt'
+
+export const up = async (db: Db, client: MongoClient) => {
+  const userMgntDb = (await new MongoClient(USER_MGNT_MONGO_URL).connect()).db()
+
+  const practitioners = await db.collection('Practitioner').find({}).toArray()
+
+  for (const practitioner of practitioners) {
```

Comment:
```suggestion
  for await (const practitioner of practitioners) {
```
we can use the `for await` syntax with the async iterator